analytics/views/support.py

```diff
@@ -119,24 +120,24 @@ def support(request: HttpRequest) -> HttpResponse:
         if len(keys) != 2:
             raise JsonableError(_("Invalid parameters"))
 
-        realm_id = request.POST.get("realm_id")
+        realm_id: str = assert_is_not_none(request.POST.get("realm_id"))
```

Comment:
This logic would probably be better off being converted to use `REQ` to parse these parameters.  @hackerkid is that something you've thought about?  We can open an issue if so.

Reply:
Yeah, I think so. This is one of the few places where we could use `REQ`. Perhaps we can do a bulk migration for the ones alike.

Reply:
Okay. Sounds good to me as long as we don't remove checks like these
```python
keys = set(request.POST.keys())
if "csrfmiddlewaretoken" in keys:
    keys.remove("csrfmiddlewaretoken")
if len(keys) != 2:
    raise JsonableError(_("Invalid parameters"))
```
@PIG208 Would you like to work on this as well or should I open an issue?

Reply:
I can do that.

Reply:
#19432